Tests/iaas/entropy/entropy-check.py

```diff
@@ -228,11 +228,11 @@ def create_vm(env, all_flavors, image, server_name=SERVER_NAME):
     # Pick a flavor matching the image
     flavors = [flv for flv in all_flavors if flv.disk >= image.min_disk and flv.ram >= image.min_ram]
     # if at all possible, prefer a flavor that provides hw_rng:allowed!
-    flavors_hrng = [flv for flv in flavors if flv.extra_specs.get("hw_rng:allowed")]
+    flavors_hrng = [flv for flv in flavors if flv.extra_specs.get("hw_rng:allowed") == "True"]
     if flavors_hrng:
         flavors = flavors_hrng
     elif flavors:
-        logger.debug(f"Unable to pick flavor with hw_rng:allowed=True for image '{image.name}'")
+        logger.debug(f"Unable to pick flavor with hw_rng:allowed=true for image '{image.name}'")
```

Comment:
if this property is a string it should also be correctly formatted in the debug message:

```python
        logger.debug(f"Unable to pick flavor with hw_rng:allowed=\"True\" for image '{image.name}'")
```

Reply:
Well, I could swear that in all the docs it's always spelled that way (`=true`), and I'm actually not sure what would be gained by using a Python-specific representation here.